### DISCUSSION TITLE
Welcome to web-dev Discussions!

### DISCUSSION THREAD

**Discussion by yigakpoa** [clentafrica/web-dev#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ayush3941** [comment]
Hey this is Ayush Kumar Gaur pursuing btech 2nd year from IET lucknow i am a full stack python developer with past experience as an on site intern in softpro india and also javascript

while i am new to open source i have designed and developed many sites like one of my design is for a startup in india is as follows:- https://www.figma.com/design/l6KDUj4Iv3NhnE210dx4y8/project-cabit?node-id=0-1&p=f&t=YhovQ0dI2EbaAQ0P-0 

coming to the topic i have visited the website and i want fix its design as currently it looks very unprofessional to be honest as shown below i need current source code of the website to come up with new design and work on it please guide me as there is no code on               clentafrica/web-dev/code or can i work on sit from scratch

**Comment by Ayush3941** [comment]
there are several very basic <div> mistakes in the following pics

![Screenshot_2025-01-20_23_30_59](https://github.com/user-attachments/assets/ca392668-5bb5-4399-baca-e262e76fe73e)
![Screenshot_2025-01-20_23_30_51](https://github.com/user-attachments/assets/74973807-9a57-4913-903d-25d8c3a1893e)
